acceptance/tests/base/dsl/helpers/web_helpers/web_helpers_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'helpers/test_helper'
+require 'webrick'
+require 'webrick/https'
+
+test_name 'dsl::helpers::web_helpers #link_exists?' do
+
+  http_cmd = "ruby -rwebrick -e'WEBrick::HTTPServer.new(:Port => 80, :DocumentRoot => \"/tmp\").start' > /tmp/mylogfile 2>&1 &"
+  master = only_host_with_role(hosts, 'master')
```

Comment:
Is this master assignment necessary? The `master` method just calls `only_host_with_role`, so I'm not sure if this is necessary.